### ISSUE TITLE
[bug-website module] Kappabeast.com no longer working in the program

### ISSUE THREAD

**Issue by Geese1** [dazedcat19/FMD#1781]
### Prerequisites

- [X] I'm using the latest version.
- [X] All website modules are up to date.
- [X] This is not temporary server issue.

### System information

Windows 11 Home 23H2 

### Describe the bug

Today I tried getting a new chapter of "The Monster Girl I Saved" from [https://kappabeast.com](https://kappabeast.com) but none of the chapters are loading into FMD. This issue was happening a little while back when they updated their website, but when FMD updated the program to 2.0.33.1 the issue was resolved. 

Now, it is happening again. 

I've tried all 3 methods of the Cloudflare workaround and none of them are having any affect. Their site is still working properly, so I don't believe it's anything on their end. 

The site is also giving me an error message when I try to view the page source saying the content is protected. 

Any help would be appreciated.

**Comment by Slasar41** [comment]
Try the manual cookies workaround. Maybe the automatic workaround is broken.
![image](https://github.com/user-attachments/assets/6d1a8775-3295-4a1a-8f21-b206b85b11a0)

**Comment by Geese1** [comment]
> Try the manual cookies workaround. Maybe the automatic workaround is broken. 

What is the manual workaround? I've searched for it here but can't find a link detailing the process.

**Comment by Slasar41** [comment]
Like [this](https://github.com/dazedcat19/FMD2/issues/187#issuecomment-736808964).

**Comment by ghost** [comment]
Tried to manually add cookies but not working.

**Comment by Slasar41** [comment]
@deckultra734 Have you tried the automatic workaround? If so, delete `use_webdriver`. If not then try with a VPN.

**Comment by Geese1** [comment]
Yeah, I'm still not having any luck with this either. I worked on this for hours yesterday trying cookies and useragents from both Chrome and Librewolf (my default browser), but nothing worked. Today I went ahead and removed all instances of Python and associated programs and then redownloaded FMD in order to get a fresh install, but that didn't work either. Also tried a VPN and still nothing. FMD can't even pull a list of titles from the Kappabeast site, so right now I have nothing in there. 

Not sure how you're able to see the chapters but some of us aren't.
